src/math/trigonometry.js

```diff
@@ -208,100 +287,105 @@ p5.prototype.sin = function(angle) {
 };
 
 /**
- * Calculates the tangent of an angle. This function takes into account
- * the current <a href="#/p5/angleMode">angleMode</a>. Values are returned in the range of all real numbers.
+ * Calculates the tangent of an angle. `tan()` is useful for many geometric
+ * tasks in creative coding. The values returned oscillate between -Infinity
+ * and Infinity. `tan()` takes into account the current
```

Comment:
I'm not sure if "oscillate between -Infinity and Infinity" gives a clear image of how the tan() function behaves. Maybe saying "the values returned range between -Infinity and Infinity" would help avoid confusion.

Reply:
Great clarification. Let me know if the updated version is more precise. Might even be too detailed.